src/banman.cpp

```diff
@@ -26,7 +26,7 @@ BanMan::BanMan(fs::path ban_file, CClientUIInterface* client_interface, int64_t
         SweepBanned();            // sweep out unused entries
 
         LogPrint(BCLog::NET, "Loaded %d banned node ips/subnets from banlist.dat  %dms\n",
```

Comment:
FWIW I would find this logging handy as a regular debug log `LogPrintf`, like the other ones in this function.